ansible/roles/test/tasks/snmp/pfc_counters.yml

```diff
@@ -6,6 +6,6 @@
 # Check PFC counters
 # Ignore management ports, assuming the names starting with 'eth', eg. eth0
 - fail:
-    msg: "Port {{ item.key }} does not has PFC counters"
-  when: (not item.value.name.startswith("eth")) and (not item.value.cpfcIfRequests or not item.value.cpfcIfIndications or not item.value.requestsPerPriority or not item.value.indicationsPerPriority)
+    msg: "Port {{ item.key }} does not have PFC counters"
+  when: (not item.value.name.startswith("eth")) and (item.value.cpfcIfRequests is not defined or not item.value.cpfcIfIndications or not item.value.requestsPerPriority or not item.value.indicationsPerPriority)
```

Comment:
leftover from my local changes.

2 check is trying to access cpfcIfRequests in item.value but it may not exists, so ansible will print an error like this: 'dict object has no cpfcIfRequests' and fail.
On the other hand 1st check with 'not defined' will print readable message: 'Port EthernetX does not have PFC counters' and fail.

I thinks we can replace these checks with 'is not defined', but not in the scope of this PR.